### DISCUSSION TITLE
make fails with error module "cmd: unknown variable '$(PATH_OVERRIDE_SOONG)'"

### DISCUSSION THREAD

**Discussion by codestheos** [community/community#20840] [Programming Help]
I want to build MTKOSP and follow the instructions here. 
https://github.com/MTKOSP/android

But it shows this error.
![image](https://user-images.githubusercontent.com/108422664/179664437-a9e762e3-441c-489a-b01a-ca484c0269e6.png)
****

Does anyone have an idea, how to resolve this issue?

**Comment by VTVishwanath** [comment]
Is this solved?
Any solution on this? I am still facing this issue while building Lineage os for RMX1851 device.

Help would be appreciated.

@codestheos

**Comment by ghostinhershell** [comment]
Hi @codestheos ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [MediaTeK Open Source Project](https://github.com/MTKOSP/android). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by wolverine2k** [comment]
Any updates on the topic? I am facing similar issue but with a different device. It is an old armv7 device that I am trying to bring up on A14.
